CodeSniffer/Standards/Squiz/Sniffs/PHP/LowercasePHPFunctionsSniff.php

```diff
@@ -119,4 +118,22 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
     }//end process()
 
 
+    /**
+     * Determine whether the functionName in question is a built-in
+     * (or extension-provided) function.
+     *
+     * @param string $functionName Name of the function to check
+     *
+     * @return bool
+     */
+    protected function isBuiltInFunction($functionName)
```

Comment:
I like the above ideas. I'll add one more. Don't make the call an additional function. It's cleaner code, but the additional function call for every T_STRING token that appears to be a function name actually adds significant overhead when checking large codebases. I know, it's horrible. But if it shaves a few seconds off a run, it's good.